docs/book/src/getting-started.md

```diff
@@ -43,11 +43,10 @@ it into a management cluster using `clusterctl`.
     ```
     - Additional varibles required for VPC
     
-    The sample `IAM_ENDPOINT` below points to Production and the `SERVICE_ENDPOINT` points to the `us-east` VPC region.
+    The sample `IAM_ENDPOINT` below points to Production.
 
     ```console
     export IAM_ENDPOINT=https://iam.cloud.ibm.com/identity/token
```

Comment:
Even this is not used anywhere, lets remove this as well.

Reply:
Done.